### PR TITLE
PDI-1615: Make SSO Connector Separate From Platform Connector

### PR DESCRIPTION
**Pull request by erikostien-pingidentity**
- Move platform connector into pingone-platform subdirectory, and
    only export platform resources with this connector
- Create new SSO connector that export SSO resources
- Use connector common package to implement DRY.
- Update custom type MultiService to handle user service
    specification on export.

### CODE REVIEW

File: internal/connector/pingone_platform/pingone_platform_connector.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		},
+	}
+}
+
+func (c *PingonePlatformConnector) Export(format, outputDir string, overwriteExport bool) error {
+	l := logger.Get()
+
+	l.Debug().Msgf("Validating export environment ID...")
+
+	environment, response, err := c.clientInfo.ApiClient.ManagementAPIClient.EnvironmentsApi.ReadOneEnvironment(c.clientInfo.Context, c.clientInfo.ExportEnvironmentID).Execute()
```
Comment (erikostien-pingidentity) [inline internal/connector/pingone_platform/pingone_platform_connector.go:44]: Look into moving this into common code so each connector doesn't do this. Perhaps move it out to export.go?

Comment (wesleymccollam) [reply internal/connector/pingone_platform/pingone_platform_connector.go:44]: We could always do it later as well. It's not a ton of repeatable code.

**Comment by wesleymccollam** [comment]
image

I think it would be good to have all products contain a parent folder (see screen snippet above). Doing this would allow us to match the Terraform registry structure of the providers.

**Review by wesleymccollam — CHANGES_REQUESTED** [review]
image

I think it would be good to have all products match their Terraform registry structure (see screen snippet above).

**Pull request merged**